### ISSUE TITLE
Bug report : Failed to connect to the remote extension host server

### ISSUE THREAD

**Issue by sumeyraunal** [microsoft/vscode-tools-for-ai#2030]
Does this occur consistently? 
Repro steps:

1.
2.

Action: Resolver.resolve
Error type: 70
Error Message: request to redacted:url failed, reason: unable to get local issuer certificate

Version: 0.28.1
OS: win32
OS Release: 10.0.19045
Product: Visual Studio Code
Product Version: 1.78.0
Language: en

<details>
<summary>Call Stack</summary>

```
s extension.js:2:1981706
extension.js:2:2007388extension.js:2:2007388
```

</details>

```[tasklist]
### Tasks
- [ ] Add a draft title or issue reference here
```
/

**Comment by sevillal** [comment]
Hi @sumeyraunal , I apologize for the issue you are facing. 

I would like to get more details about your scenario:
- Is your workspace secured with Private Link and you compute instance is inside a Vnet?
- Is the compute instance failing to connect recently created?
- Were you able to connect to the failing compute instance before?
- Were you able to connect to other compute instances before?
- Can you try connecting to that compute instance from a different machine? (just in case you have access to another computer)

**Comment by sevillal** [comment]
@sumeyraunal also, could you please try the following?

Install this extension to diagnose problems like this: https://marketplace.visualstudio.com/items?itemName=chrmarti.network-proxy-test. Then use F1 > Network Proxy Test: Test Network Connection with the URL is failing (I assume it might be something like https://<CI-NAME>.<REGION>.instances.azureml.ms/api/terminals) to check if that works. If it shows the certificate error retry with F1 > Network Proxy Test: Test Network Connection (Allow Unauthorized) to get more information about the problem.

Make sure replace any sensitive information before posting it here. Thanks.

**Comment by sevillal** [comment]
@sumeyraunal VS Code just released version 1.78.2 which should fix the certificates issue, could you please update VS Code (Help > Check for Updates...) and retry the connection.

**Comment by sevillal** [comment]
Closing it since other customers have confirmed VS Code version 1.78.2 corrects the issue. Please reopen if not fixed.
